frontend/src/components/ViewParty/ViewParty.js

```diff
@@ -10,6 +10,28 @@ import axios from 'axios';
 
 const ViewParty=(props)=>{
 
+  const joinParty=() =>{
+
+    const headers = {
+      headers: {
+        "Content-Type": "application/json",
+      },
+    };
+  
+  const Link = `${process.env.REACT_APP_URL}/Party/${props.ID}/Update`;
```

Comment:
That link has one too many '/'s. replace it with this:
const Link = `${process.env.REACT_APP_URL}Party/${props.ID}/Update`;